cmd/get-hardware-details/main.go

```diff
@@ -58,7 +59,7 @@ func main() {
 
 func getOptions() (o options) {
 	if len(os.Args) != 3 {
-		fmt.Println("Usage: get-hardware-details <inspector URI> <node UUID>")
+		fmt.Println("Usage: get-hardware-details <ironic URI> <node UUID>")
```

Comment:
This looks like a backwards-compat nightmare :(

Could we do some sort of temporary hack where we the substitute the well-known ironic port number for the well-known inspector port number when it appears maybe?

Reply:
I wonder if this tool is even used (I'd rather have people use the `baremetal` CLI from Ironic, sigh). I'll see if I can invent something.

Reply:
I think it's used in the [OpenShift installer](https://github.com/openshift/installer/blob/master/data/data/bootstrap/baremetal/files/usr/local/bin/master-bmh-update.sh#L45) ;)

Reply:
Yep, it's definitely used by us, but _probably_ nobody else.